PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetQnVectors.h

```diff
@@ -93,6 +88,7 @@ class AliAnalysisTaskJetQnVectors : public AliAnalysisTaskSE {
     unsigned long long fTriggerMask;                 /// trigger mask
 
     double fq2V0;                                    /// q2 vector from the V0     
+    double fEPangleCF;                               /// EP Angle with callibrations from V0
```

Comment:
You should initialize this in the constructor. And since you've implicitly set this to be a streaming (I think - it's a bit ambiguous), you should bump the ClassDef